.babelrc

```diff
@@ -1,11 +1,10 @@
 {
   "presets": [
-    ["env", {
+    ["@babel/preset-env", {
         "targets": {
           "browsers": ["last 1 versions", "ie >= 10"]
```

Comment:
We might want to set this to lower than last 1 versions, also, might be nice to have a node v10+ or something in there.